app/assets/javascripts/application.js

```diff
@@ -17,3 +17,6 @@
 //= require rails-ujs
 //= require turbolinks
 //= require_tree
+//= require jquery
+//= require bootstrap
```

Comment:
Try not to import jquery and bootstrap multiple times. It won't break anything but is repetitive.